### ISSUE TITLE
Cannot run the help command

### ISSUE THREAD

**Issue by shwuhk** [Unity-CI/docker#42]
Thanks for creating this great project!
I tried to start the editor as stated by the doc using the following commands but it gives me an error message. I tried to put the license file at `/root/.local/share/unity3d/Unity/Unity_lic.ulf` but still no luck.

Commands:
```
docker run -it --rm unityci/editor:2019.2.12f1-android-0 bash
unity-editor help
```

Error Message:
```
[1108/204733:ERROR:browser_main_loop.cc(161)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
[1108/204733:ERROR:sandbox_linux.cc(308)] InitializeSandbox() called with multiple threads in process gpu-process
```

I am not sure if this is a bug or just I don't know how to use it. Any help is appreciated! Thanks.

**Comment by webbertakken** [comment]
The help command may not work, but other commands do. As the focus of this project is primarily to support CI pipelines, and we offer wide range of documentation on [how to](https://unity-ci.com/docs) set that up, priority for fixing the help command is not very high.

Please do feel free to add more comments if you have any clues.

**Comment by shwuhk** [comment]
It is because I tried to run unity-editor and it also give me this error message, so I just thought it just not gonna work.

**Comment by webbertakken** [comment]
Multiple successful setups with Android builds have already been reported on our [discord](http://unity-ci.com/discord) channel, these new images are working for the most part. Just not the help command. And I believe without any arguments the CLI has never worked.

By the way [Unity-Builder](https://github.com/marketplace/actions/unity-builder) also successfully implements the new images in v2, so you could possibly save yourself a lot of time, by not implementing all of that yourself.

Cheers!

**Comment by shwuhk** [comment]
Thanks for your help and this great project again!
Let me try again with arguments!
